schema_based_on_diagram.sql

```diff
@@ -0,0 +1,63 @@
+/* Creating tables for clinic database and primery keys*/
+
+create table patients(
+    id int generated always as identity,
+    name varchar(60),
+    date_of_birth date,
+    primary key(id)
+)
+
+create table medical_histories(
+    id int generated always as identity,
+    admitted_at timestamp,
+    patient_id int,
+    status varchar(60),
+    primary key(id)
+);
+
+create table invoices(
+    id int generated always as identity,
+    total_amount decimal,
+    generated_at timestamp,
+    payed_at timestamp,
+    medical_history_id  int,
+    primary key(id)
+);
+
+create table invoices_items(
+    id int generated always as identity,
+    unit_price decimal,
+    quantity  int,
+    total_price  decimal,
+    invoice_id  int,
+    treatment_id int,
```

Comment:
- [ ] Kindly notice that it is best to add the reference to the foreign key in the creation of the table.

```sql
invoice_id  int reference invoices(id),
treatment_id int reference treatments(id),
```